simple_graph.py

```diff
@@ -0,0 +1,60 @@
+class Graph(object):
+
+    def __init__(self):
+        self.graph = {}
+
+    def nodes(self):
+        l = []
+        for k, v in self.graph.iteritems():
+            l.append(k)
+        return l
```

Comment:
why iterate over the items if all you want is the keys?  Is there no way to get just the keys by themselves without doing all this work?